cost/azure/databricks/rightsize_compute/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
```

Comment:
@JesusParada One last thing - regarding this error:
<blockquote>
cost/azure/databricks/rightsize_compute/azure_databricks_rightsize_compute.pt
Policy Template file has not yet been enabled for automated permission generation. Please help us improve coverage by [following the steps documented in tools/policy_master_permission_generation/](https://github.com/flexera-public/policy_templates/tree/master/tools/policy_master_permission_generation) to resolve this.
--
</blockquote>

Is it possible to add the Azure Databricks Rightsize Compute policy template to the validated policy list here? - https://github.com/flexera-public/policy_templates/blob/3983e1dd7269a7e2f1811dfa997012392e940f04/tools/policy_master_permission_generation/validated_policy_templates.yaml (under the Azure section following alphabetical order)